doc/7/getting-started/.react/standalone/src/App.js

```diff
@@ -0,0 +1,166 @@
+/* snippet:start:1 */
+import React from 'react';
+import kuzzle from "./services/kuzzle";
```

Comment:
Nitpicking: string must use singlequotes
```suggestion
import kuzzle from './services/kuzzle';
```